modules/api/current.js

```diff
@@ -234,10 +238,17 @@ module.exports = {
       testData.parameters = this.storage.getParameters(parametersId);
     }
 
+    const { children } = feature;
+    const stepsAstNodesData = utils.findAstNodesData(children);
+    const astNodesDataField = Symbol.for(SYMBOLS_NAMES.astNodesDataField);
+
     const parentId = ruleTempId || this.storage.getFeatureTempId();
     const { tempId } = this.reportportal.startTestItem(testData, launchTempId, parentId);
     this.storage.setScenarioTempId(tempId);
-    this.storage.updateTestCase(testCaseId, { codeRef: scenarioCodeRef });
+    this.storage.updateTestCase(testCaseId, {
+      codeRef: scenarioCodeRef,
+      [astNodesDataField]: stepsAstNodesData,
```

Comment:
The main idea of using Symbols here is to prevent reassigning that property. Because we change a lot of properties by link, and that object can be processed by Cucumber. And it is recommended to use Symbols in that case.

Reply:
The main reason for using Symbol here is to prevent reassigning that property. We change a lot of different properties by link, and that object can be processed by Cucumber. I see that testCase has only three properties, but if tomorrow developers of Cucumber decided to add such field in that object.